### ISSUE TITLE
[Bug]: Network connection breaks when on wired instead of wifi

### ISSUE THREAD

**Issue by bobziuchkovski** [PlayCover/PlayCover#952]
### Describe the bug

I have a Macbook Pro with an M1 Max. I'm using PlayCover to run Netflix and everything works swimmingly while I'm on WiFi, but the moment I plug-in to my dock (which provides a Thunderbolt ethernet connection), the app reports there's no network connection. This keeps occurring regardless of exiting/restarting the app/PlayCover until I explicitly disable the Thunderbolt connection ("Make Inactive" button in Network settings). Once I do that and the WiFi connection becomes the primary connection, Netflix is happy / stops throwing errors about my connectivity.

The thunderbolt ethernet connection definitely -is- connected to the internet and everything else outside of PlayCover/Netflix are happy with it. I have nothing strange like any proxies/VPNs active or anything. I've even tried explicitly turning off my Wifi connection so that -only- the thunderbolt connection is active. Everything else on my system keeps trucking / using the internet as usual but PlayCover/Netflix believe I'm offline.

### Steps to reproduce

Open Netflix in PlayCover. Attach a thunderbolt network adapter/dock to the Macbook. Connect the adapter to the internet. Try to use Netflix in PlayCover.

### Expected behaviour

I'd expect PlayCover/Netflix to think I'm online when I'm connected to the internet via thunderbolt ethernet. 

### Crash log

No crashes. I tried to find relevant logs but couldn't locate them. I'm happy re-test and provide logs if I know where to look for them.

### What version of PlayCover are you using?

2.0.5

### What version of macOS are you using?

Ventura (macOS 13)

### Issue Language

- [X] Yes my issue is written in English

**Comment by ohaiibuzzle** [comment]
Makes sense, the issues is that these apps kinda expects Cellular or WiFi and not Ethernet
So if they are programmed in such a way where other connection types aren't considered they would think there isn't any

**Comment by bobziuchkovski** [comment]
Hmm...that makes sense. It isn't that big a deal since I can just use WiFi, but would be neat if there were a way to present a sort of virtual wifi interface that represents having any sort of network connection from your Mac. 🤷

**Comment by ohaiibuzzle** [comment]
> Hmm...that makes sense. It isn't that big a deal since I can just use WiFi, but would be neat if there were a way to present a sort of virtual wifi interface that represents having any sort of network connection from your Mac. 🤷

Problem with doing so is that it would make it trivial for apps to detect PlayCover's presence
You know how Android emulators all have their Wi-Fi name like "VirtualWiFi" and such? Yeah, took apps *not so long* to figure that out
